scripts/qa_crowbarsetup.sh

```diff
@@ -2032,12 +2032,11 @@ function onadmin_proposal()
     if iscloudver 5plus; then
         update_one_proposal dns default
     fi
-    if [ "$networkingplugin" = "vmware" ] && iscloudver 5plus ; then
```

Comment:
@aspiers could you please add to the commit message, why you removed this check? it just says it should always happen